### ISSUE TITLE
Screen reader will focus divider on breadcrumb

### ISSUE THREAD

**Issue by Anschur** [OfficeDev/office-ui-fabric-react#8119]
### Environment Information 
 
- __Package version(s)__: https://developer.microsoft.com/en-us/fabric#/components/breadcrumb
- __Browser and OS versions__: (fill this out if relevant)

### Please provide a reproduction of the bug in a codepen:
 

 
1. Go to https://developer.microsoft.com/en-us/fabric#/components/breadcrumb
2. Start Narrator and go to a bread crumb
3. Enable scan-mode in Narrator (CapsLk + Space to toggle)
4. Arrow down through the breadcrumb elements

#### Actual behavior:
 
Narrator will focus every divider > in the list, but will say nothing about it.
 
#### Expected behavior:
 
I see no purpose of focusing the dividers. They should be hidden from the screen reader by using aria-hidden="true"

#### Priorities and help requested:
 
Are you willing to submit a PR to fix? No
 
Requested priority: Normal
 
Products/sites affected: (if applicable)

**Comment by Anschur** [comment]
Hm. I hear that if it is a slash that is a divider then it will actually be read. So the right fix might not be to hide it, but always have it read something for the divider.

**Comment by leddie24** [comment]
Hi @Anschur , sorry for the late response here.  After chatting with @betrue-final-final on this issue, he concluded that this is actually by design of narrator.  This issue is related to #8139, which I'm looking into a solution for at the moment.

**Comment by leddie24** [comment]
Removing author feedback tag.  If we're adding aria-hidden to presentational icons then pr #8381 fixes the custom divider to be consistent in behavior.
